packages/curve-ui-kit/src/widgets/Footer/Section.tsx

```diff
@@ -0,0 +1,34 @@
+import Grid from '@mui/material/Grid2'
+import Typography from '@mui/material/Typography'
+
+import { SizesAndSpaces } from 'curve-ui-kit/src/themes/design/1_sizes_spaces'
+
+import { Link, LinkProps } from './Link'
+
+export type SectionProps = {
+  title: string
+  links: LinkProps[]
+}
+
+export const Section = ({ title, links }: SectionProps) => (
+  <Grid container spacing={1}>
+    <Grid size={12}>
+      <Typography
+        variant="headingXsBold"
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          height: SizesAndSpaces.ButtonSize.sm,
+        }}
+      >
+        {title}
+      </Typography>
+    </Grid>
+
+    {links.map((link, i) => (
+      <Grid key={i} size={{ mobile: 6, tablet: 12, desktop: 12 }}>
```

Comment:
```suggestion
      <Grid key={i} size={{ mobile: 6, desktop: 12 }}>
```

Reply:
Heh, actually had to remove `desktop: 12` so that it's still 12 in tablet as well.